documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -349,17 +349,16 @@ The {scope} is the specific scope defined to protect this operation.
 
 ### Mandatory template for `info.description` in CAMARA API specs
 
-The documentation template below must be used as part of the API documentation in  `info.description` property in the CAMARA API specs:
-
+The documentation template below must be used as part of the API documentation in `info.description` property in the CAMARA API specs:
 
 ```
 ### Authorization and authentication
 
-The "Camara Security and Interoperability Profile" provides details on how a client requests an access token. Please refer to Identify and Consent Management (https://github.com/camaraproject/IdentityAndConsentManagement/) for the released version of the Profile.
+The "Camara Security and Interoperability Profile" provides details on how a client requests an access token.
 
-Which specific authorization flows are to be used will be determined during onboarding process, happening between the API Client and the Telco Operator exposing the API, taking into account the declared purpose for accessing the API, while also being subject to the prevailing legal framework dictated by local legislation.
+The specific authorization flows to be used will be determined during onboarding process, happening between the API client and the telco operator exposing the API, taking into account the declared purpose for accessing the API, whilst also being subject to the prevailing legal framework dictated by local legislation.
```

Comment:
```suggestion
The specific authorization flows to be used will be determined during the onboarding process, happening between the API client and the operator exposing the API, taking into account the declared purpose for accessing the API, whilst also being subject to the prevailing legal framework dictated by local legislation.
```

Can we drop telco, since these APIs can be provided by MSOs and other non-telco operators?

Also the missing "the" was in my initial comment.